<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EdytujWBazie</class>
 <widget class="QDialog" name="EdytujWBazie">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTextEdit" name="teMarka">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teModel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teNrRej">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="teVin">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbEdytuj">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Edytuj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAnuluj">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>100</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Anuluj</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Marka:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Model:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>NrRej:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>vin:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
